Example/Example/AppDelegate.swift

```diff
@@ -271,12 +271,42 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIWindowSceneDelegate, UN
         }
 
         demoButton(text: "startTrip") {
-            let tripOptions = RadarTripOptions(externalId: "299", destinationGeofenceTag: "store", destinationGeofenceExternalId: "123")
+            let tripOptions = RadarTripOptions(externalId: "300", destinationGeofenceTag: "store", destinationGeofenceExternalId: "123")
```

Comment:
nit: this may cause an annoyance for manual testing, I personally use UUIDS for these external IDs to avoid the server complaining about external ID collision.